.jscsrc

```diff
@@ -1,51 +1,78 @@
 {
-    "disallowEmptyBlocks": true,
-    "disallowMixedSpacesAndTabs": "smart",
-    "disallowNewlineBeforeBlockStatements": true,
-    "disallowSpaceAfterObjectKeys": true,
-    "disallowSpaceBeforePostfixUnaryOperators": true,
-    "disallowSpacesInFunction": {
-        "beforeOpeningRoundBrace": true
-    },
-    "disallowSpacesInCallExpression": true,
-    "disallowSpacesInsideArrayBrackets": true,
-    "disallowSpacesInsideParentheses": true,
-    "disallowTrailingComma": true,
-    "disallowTrailingWhitespace": true,
-    "requireBlocksOnNewline": 1,
-    "requireCamelCaseOrUpperCaseIdentifiers": true,
-    "requireCapitalizedConstructors": true,
-    "requireCommaBeforeLineBreak": true,
-    "requireCurlyBraces": true,
-    "requireDotNotation": true,
-    "requireLineBreakAfterVariableAssignment": true,
-    "requirePaddingNewlinesBeforeKeywords": true,
-    "requireParenthesesAroundIIFE": true,
-    "requireSpaceAfterBinaryOperators": true,
-    "requireSpaceAfterKeywords": [
+    "requireCurlyBraces": [
         "if",
         "else",
         "for",
         "while",
         "do",
-        "switch",
-        "case",
-        "return",
         "try",
         "catch"
     ],
-    "requireSpaceAfterLineComment": true,
-    "requireSpaceBeforeBinaryOperators": true,
-    "requireSpaceBeforeBlockStatements": true,
-    "requireSpaceBeforeKeywords": [
-        "else",
-        "while",
-        "catch"
+    "requireOperatorBeforeLineBreak": true,
+    "requireCamelCaseOrUpperCaseIdentifiers": true,
+    //"maximumLineLength": {
```

Comment:
I'd argue, even if 80 characters is too hard to keep to, that there's some enforced hard limit on line length. Anything longer than 120 characters and you run into the following problems across the board:
- Diff issues with multiple declarations / statements on the same line being harder to merge
- Having to scroll horizontally in editor to view/edit file
- Visibility of code